apps/wing/src/commands/pack.test.ts

```diff
@@ -136,6 +136,45 @@ describe("wing pack", () => {
     expect(pkgJson.wing).toEqual(true);
   });
 
+  it("can consume a Wing project from JS", async () => {
+    // GIVEN
+    const outdir = await generateTmpDir();
+    // copy everything to the output directory to sandbox this test
+    await exec(`cp -r ${goodFixtureDir}/* ${outdir}`);
+    process.chdir(outdir);
+
+    // WHEN
```

Comment:
Is it possible to also add an example of a javascript program that uses a wing library?

Reply:
Currently, the compiler explicitly expects a `.ts` entrypoint. It could technically allow `.js` but I'm not sure there's much value adding and testing that yet.